src/experiments/example-experiment/index.tsx

```diff
@@ -6,7 +6,7 @@ import { __ } from '@wordpress/i18n';
 import './index.scss';
 
 const ExampleExperiment = () => {
-	return <div className="test">{ __( 'Example Experiment', 'ai' ) }</div>;
+	return <div className="test">{__('Example Experiment', 'ai')}</div>;
```

Comment:
Why this "correction"? Before it was using WordPress Coding Standards, and now it isn't.

Reply:
That is really strange. I don't know if there was a recent update, but I've been using `wp-scripts` myself and it would always enforce the opposite (I.e. WordPress Coding Standards). Is there something else at play here maybe that leads to the regular Prettier taking over the WordPress specific version? 🤔 

This needs some more digging because that's not something that should be changed without broader consensus by the Core developers. A few years ago there was actually a proposal to change it which ended up rejected.

Reply:
Is there maybe a conflicting version of `prettier` installed that's _not_ the `wp-prettier` fork? If some dependency uses `prettier`, maybe that's what's happening.

[`@wordpress/scripts` uses `wp-prettier`](https://github.com/WordPress/gutenberg/blob/trunk/packages/scripts/package.json#L78), so that's all as I thought it would be. But maybe another dependency is pulling in the "real" `prettier`, and that might cause the problem?